server/utils/README.md

```diff
@@ -0,0 +1,7 @@
+# **API call to fetch 311 request data from the 311 data server for a given start date and end date.** 
```

Comment:
This directory will serve as a location for multiple utility tools, so please note that first. Then, we can have different sections describing each tool.

Reply:
Sounds good! Will add this information.